### PR TITLE
Select destination place

### PR DESCRIPTION
**Pull request by dwantula**

### CODE REVIEW

File: src/screens/Home/Home.jsx [hunk 1]
```diff
@@ -2,23 +2,35 @@ import React, { useEffect, useState } from 'react';
...
 
 function Home() {
   const [chosenCountry, setChosenCountry] = useState('');
+  const [chosenPlace, setChosenPlace] = useState('');
 
   const { countries } = useSelector((state) => state.countries);
   const isLoadingCountries = useSelector((state) => state.countries.loading);
+  const { places } = useSelector((state) => state.places);
+  const isLoadingPlaces = useSelector((state) => state.places.loading);
+  console.log(places);
```
Comment (mi-chal) [inline src/screens/Home/Home.jsx]: Remove console.log

File: src/screens/Home/Home.jsx [hunk 2]
```diff
@@ -2,23 +2,35 @@ import React, { useEffect, useState } from 'react';
...
     dispatch(getCountriesAction());
   }, [dispatch]);
 
+  useEffect(() => {
+    if (chosenCountry) {
+      dispatch(getPlacesAction(chosenCountry));
+    }
+  }, [dispatch, chosenCountry]);
+
+  console.log(chosenPlace);
```
Comment (mi-chal) [inline src/screens/Home/Home.jsx]: Remoce console.log

File: src/screens/Home/Home.jsx [hunk 3]
```diff
@@ -32,6 +44,15 @@ function Home() {
           inputPlaceholder="Search countries"
           isLoadingItems={isLoadingCountries}
         />
+        {chosenCountry && (
```
Comment (mi-chal) [inline src/screens/Home/Home.jsx]: Use ternary instead of `&&`

File: src/screens/Home/styles.scss [hunk 4]
```diff
@@ -9,6 +9,11 @@
   background-size: 100% 100%;
 }
 
+// .main--row {
+//   display: flex;
+//   flex-direction: row;
+// }
+
```
Comment (mi-chal) [inline src/screens/Home/styles.scss]: Remove comments

**Review by mi-chal — APPROVED** [review]

File: src/screens/Home/Home.jsx (hunk 1 above)
Comment (dwantula) [reply src/screens/Home/Home.jsx]: Done

File: src/screens/Home/Home.jsx (hunk 2 above)
Comment (dwantula) [reply src/screens/Home/Home.jsx]: Done!

File: src/screens/Home/Home.jsx (hunk 3 above)
Comment (dwantula) [reply src/screens/Home/Home.jsx]: Fixed

File: src/screens/Home/styles.scss (hunk 4 above)
Comment (dwantula) [reply src/screens/Home/styles.scss]: Done

**Pull request merged**